lib/src/main/java/com/auth0/jwt/impl/JsonNodeClaim.java

```diff
@@ -91,6 +92,18 @@ public Date asDate() {
     }
 
     @Override
+    public Map<String, Object> asMap() throws JWTDecodeException {
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
It's on our backlog to enhance this. We're thinking on moving the implementation to use Jackson's `Reader` and `Writer`.